demos/golang/go_sqlite/run_go_sqlite_demo.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+set -e
+
+BLUE='\033[1;34m'
+NC='\033[0m'
+
+# Install SQLite with occlum-go
+occlum-go get -u -v github.com/mattn/go-sqlite3
+
+# Build the Golang SQLite demo program using the Occlum Golang toolchain(i.e., occlum-go)
+occlum-go build -o simple_demo simple_demo.go
+
+sqlite_demo="./simple_demo"
+if [ ! -f $sqlite_demo ];then
```

Comment:
I don't think this check is necessary. If the `occlum-go build` command failed, then the script should have exited (due to `set -e`).

Reply:
Done. That is great suggestion. Thanks.